### ISSUE TITLE
install ipa crash on ios 14.1  version 1.4.1

### ISSUE THREAD

**Issue by nap-liu** [rileytestut/AltStore#416]
**Describe the bug**
install ipa crash on ios 14.1  version 1.4.1

**To Reproduce**
1. download xcodebuild command generated ipa 
2. altstore install ipa
3. start progress and crashed

**iPhone (please complete the following information):**
 - Device: iphone11
 - iOS: 14.1

**Additional context and logs**
Hardware Model:      iPhone12,1
Process:             AltStore [498]
Path:                /private/var/containers/Bundle/Application/E68B2AFB-A1D2-424C-93F8-1274E5DC438C/AltStore.app/AltStore
Identifier:          com.6EPJ9E3C59.com.rileytestut.AltStore
Version:             1 (1.4.1)
Code Type:           ARM-64 (Native)
Role:                Foreground
Parent Process:      launchd [1]
Coalition:           com.6EPJ9E3C59.com.rileytestut.AltStore [517]

Date/Time:           2020-11-02 13:57:37.2553 +0800
Launch Time:         2020-11-02 13:57:27.7691 +0800
OS Version:          iPhone OS 14.1 (18A8395)
Release Type:        User
Baseband Version:    2.01.05
Report Version:      104

Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Triggered by Thread:  1

Thread 1 name:  Dispatch queue: com.apple.root.default-qos
Thread 1 Crashed:
0   libsystem_kernel.dylib        	0x00000001b4c9898c 0x1b4c71000 + 162188
1   libsystem_pthread.dylib       	0x00000001d0a26c74 0x1d0a24000 + 11380
2   libsystem_c.dylib             	0x0000000191c6ebb4 0x191bf8000 + 486324
3   AltStore                      	0x000000010223ca20 0x102100000 + 1296928
4   AltStore                      	0x0000000102224100 0x102100000 + 1196288
5   AltStore                      	0x000000010222f068 0x102100000 + 1241192
6   libc++abi.dylib               	0x000000019cd232a8 0x19cd14000 + 62120
7   libc++abi.dylib               	0x000000019cd23234 0x19cd14000 + 62004
8   libdispatch.dylib             	0x0000000188734adc 0x188731000 + 15068
9   libdispatch.dylib             	0x0000000188737a10 0x188731000 + 27152
10  libdispatch.dylib             	0x0000000188745104 0x188731000 + 82180
11  libdispatch.dylib             	0x00000001887458e8 0x188731000 + 84200
12  libsystem_pthread.dylib       	0x00000001d0a278cc 0x1d0a24000 + 14540
13  libsystem_pthread.dylib       	0x00000001d0a2e77c 0x1d0a24000 + [42876](url)

Thread 1 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000000000010   x5: 0x0000000000000020   x6: 0x0000000000000000   x7: 0x0000000000000000
    x8: 0x00000000000005b9   x9: 0xa69ae699107af54e  x10: 0x00000001d694b8f0  x11: 0x000000000000001f
   x12: 0x0000000000bb4700  x13: 0x01000001e6b8d461  x14: 0x0000000000000004  x15: 0x00000001e6b8d460
   x16: 0x0000000000000148  x17: 0x000000016dd87000  x18: 0x0000000000000000  x19: 0x0000000000000006
   x20: 0x0000000000001707  x21: 0x000000016dd870e0  x22: 0x0000000000000000  x23: 0x0000000000000114
   x24: 0x000000016dd870e0  x25: 0x0000000000000000  x26: 0x00000000000005ff  x27: 0x00000001e6b8c3c0
   x28: 0x0000000002ffffff   fp: 0x000000016dd86950   lr: 0x00000001d0a26c74
    sp: 0x000000016dd86930   pc: 0x00000001b4c9898c cpsr: 0x40000000
   esr: 0x56000080  Address size fault

**Comment by halfbread10** [comment]
Having the same issue. iPhone X 14.1 AltStore 1.4.1.

**Comment by telepati** [comment]
I have the same issue. This is happening when trying to install bigger .ipas like up to 1GB or 2 GB. All small sideloaded .ipa can easily install without any problem. When trying to big ipas Altstore just crashing.

[AltStore-2020-12-16-194013.zip](https://github.com/rileytestut/AltStore/files/5704142/AltStore-2020-12-16-194013.zip)

**Comment by shanegillio** [comment]
hi there! sorry for the lack of response - are you still experiencing issues?

**Comment by telepati** [comment]
Yes. The bigger size ipa still has problems while trying to install.

**Comment by shanegillio** [comment]
what version of AltServer/AltStore are you trying this on now?

**Comment by telepati** [comment]
I am using v1.5.1. For example, I am trying to install the El Hijo game, which is 1.175 GB. It gives an error the app is Invalid with Altserver/Altstore. But when I try to install the same app with Sideloading app no problem.
